feature-libs/product-configurator/rulebased/components/attribute/types/checkbox/configurator-attribute-checkbox.component.ts

```diff
@@ -27,6 +27,7 @@ export class ConfiguratorAttributeCheckBoxComponent
   attributeCheckBoxForm = new FormControl('');
 
   ngOnInit() {
+    console.log(this.attribute, 'checkbox');
```

Comment:
remove `console.log`

Reply:
it is GH issue :) in code we don't have a console log 
![Zrzut ekranu 2021-01-13 090757](https://user-images.githubusercontent.com/12393718/104423818-db64f900-557e-11eb-828c-f2c9067c65ab.png)